server/db/models/User.js

```diff
@@ -43,6 +43,15 @@ module.exports = function (sequelize, DataTypes) {
         });
       },
 
+      getEmployeeEmail: function (id) {
```

Comment:
we should probably just add a parameter to getEmployeeInfo which accepts an array of strings with properties on them. that can be in a refactor later on